chiselFrontend/src/main/scala/chisel3/Annotation.scala

```diff
@@ -1,13 +1,13 @@
 // See LICENSE for license details.
 
-package chisel3.core
+package chisel3.experimental
```

Comment:
I did this initially (moving experimental definitions into their own files in the experimental folder) but reverted it because:
- experimental items inheriting from sealed classes need to be in the same file as the parent class,
- `diff`ing code that's split into multiple files is problematic
- merging code that's been split into multiple files is even more problematic

I think it's a fine idea to split independent components into individual files once this PR is accepted.